ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/patch-panel.j2

```diff
@@ -0,0 +1,29 @@
+{% if patch_panel is arista.avd.defined %}
+
+## Patch Panel
```

Comment:
This should only have one `#`. And the others below accordingly.
Missing config section like we normally have.

Reply:
Fixed.